src/get-started/extending-and-modifying-components/index.md

```diff
@@ -28,6 +29,7 @@ You can help reduce potential risk to your code by:
 - creating [custom override classes for multiple components](#custom-override-classes)
 - using BEM for [small modifications to components](#small-modifications-to-components)
 - forking components when you are making [large modifications](#large-modifications-to-components)
+- checking [accessibility](#check-accessibility)
```

Comment:
```suggestion
- [checking for accessibility](#check-for-accessibility)
```

Reply:
@calvin-lau-sig7, I thought it was intentional that none of these include all the necessary words in the links, so I copied that style. I assumed that was due to Target Size (Minimum) to not have a list item that is only a link and doesn't contain any non-links?

Reply:
I wasn't thinking about Target size...

It's more that just saying "accessibility" on its own doesn't seem that meaningful? So I added a verb to at least make it clear what needs to be done.